set.mm

```diff
@@ -424459,6 +424480,51 @@ then they have the same degree (i.e., the same number of adjacent
       BVSURUSUTVAVBUNSWJWAVFWJVQWAVQWIULTWSRVCVDWKDVGJZWGPZWOWMOWJXFWAVQXEWIWGD
       VOZWTVHTVSDEFVIVJWKXEXAPZWQWNOWJXHWAVQXEWIXAXGXBVHTVRDEFVIVJVKVSCWLGVLVRC
       WLGVLVMVPVN $.
+
+    $d D x y $.  $d X x y $.  $d Y x y $.
+    frgrwopreglem4a.e $e |- E = ( Edg ` G ) $.
+    $( In a friendship graph each vertex is connected with a vertex with a
+       different degree.  Alternate version of ~ frgrwopreglem4 without a fixed
```

Comment:
The statement looks more powerful: "...each vertex is connected to **any** vertex of different degree" or maybe better "any two vertices with different degrees are connected".